samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -297,3 +300,101 @@ SnapToCenterVirtualGridList.storyName = 'Snap to center VirtualGridList';
 SnapToCenterVirtualGridList.parameters = {
 	propTables: [Config]
 };
+
+// eslint-disable-next-line enact/prop-types
+const renderItemWithRandomSrc = ({index, ...rest}) => {
```

Comment:
Can we just use `renderItem` above?

Reply:
I was confused, we can reuse `renderItem`. It's fixed.